test/cmdlineTests/evmasm_transient_storage_opcodes/output

```diff
@@ -0,0 +1,35 @@
+
+======= evmasm_transient_storage_opcodes/input.sol:C =======
+EVM assembly:
+  0x80
+  dup1
+  0x40
+  mstore
+  jumpi(tag_1, callvalue)
+  dataSize(sub_0)
+  swap1
+  dup2
+  dataOffset(sub_0)
+  dup3
+  codecopy
+  return
+tag_1:
+  0x00
+  dup1
+  revert
+stop
+
+sub_0: assembly {
+      jumpi(tag_1, callvalue)
+      0x00
+      dup1
+      tstore
```

Comment:
The value is unused, I assume it gets optimized out.

I mean, we could do something like `sstore(0, tload(0))` so that it does not disappear, but not having this also shows that the optimizer works, so for me either is fine. We could also do both.

Reply:
I'd assume so too, but having a test for `tstore` and `tload` where `tload` gets optimized away is not ideal. I think your suggestion with `sstore(0, tload(0))` would ideal.

Reply:
It shows that `tload`, when unused, is optimized away just like `sload` would be. But fine adding the `sstore(0, tload(0))` instead.

Reply:
Yup, but the `fullSuite` test is better suited for that (which you've already added) - especially since this is the only test we have that shows the opcodes.